eZ/Publish/Core/Repository/Tests/Service/Integration/RoleBase.php

```diff
@@ -575,6 +575,33 @@ public function testLoadRoles()
     }
 
     /**
+     * Test loading user/group role assignments.
```

Comment:
These tests should be in the API test case instead of here, this test case is actually deprecated and to be merged with API integration tests.